_topics/en/channel-commitment-upgrades.md

```diff
@@ -36,9 +36,9 @@ optech_mentions:
     url: /en/newsletters/2022/04/06/#updating-ln-commitments
 
 ## Optional.  Same format as "primary_sources" above
-# see_also:
-#   - title:
-#     link:
+see_also:
+  - title: Anchor outputs
+    link: topic anchor outputs
 
```

Comment:
Perhaps LN Symmetry should be linked as a related topic here as well, since it essentially is a replacement of the commitment scheme?

```suggestion

  - title: "LN-Symmetry (Eltoo)"
    link: topic eltoo
```

Reply:
Indeed!